src/Components/sortdescription/sortdescription.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './sortdescription.style.scss';
+
+const SortDescriprion = ({ data, chosenMovie }) => {
+    const correctMovie = data.filter(({ idFilm }) => idFilm === chosenMovie);
+
+    return (
+        <div className="sortDescriprionContainer">
+            <div className="sortDescriprionBy">{`Films by ${correctMovie[0].genresFilm.slice(
+                0,
+                1
+            )} genre`}</div>
+        </div>
+    );
+};
```

Comment:
Why do we need it? We need to display  only genre. This component has to receive genre from above and render it. You can use whole object of chosen movie instead id, and you will not need search this movie in the list every time.